core/sql/executor/ExExeUtilCommon.cpp

```diff
@@ -641,8 +649,12 @@ short ExExeUtilTcb::executeQuery(char * task,
 	    if (displayEndTime)
 	      {
 		char timeBuf[200];
```

Comment:
Perhaps you want to delete this declaration, since it looks like you moved it up to line 558 in the new file?